float32 pitch
float32 yaw
float32 pitchSpeed
float32 yawSpeed

